packages/teleport-plugin-react-proptypes/src/index.ts

```diff
@@ -37,10 +37,15 @@ export const createPlugin: ComponentPluginFactory<ReactJSPropTypesConfig> = (con
     }
 
     // TODO used the name from the mappings of the component, not from the UIDL
-    const defaultPropsAst = buildDefaultPropsAst(name, uidl.propDefinitions)
+    const hasDefaultProps = Object.keys(uidl.propDefinitions).some(
+      (prop) =>
+        uidl.propDefinitions[prop].defaultValue &&
```

Comment:
Only in this case we need to check becasue of the this behaviour
~~~
typeof undefined !== undefined // true
~~~

This returns ``true`` which in returns marks the flag as ``hasDefaultProps`` to ``true`` and an empty tag gets generated.

Reply:
ah, I missed this, it should be `typeof uidl.propDefinitions[prop].defaultValue !== 'undefined'` and that covers all cases

Reply:
Done 👍